packages/p2p/src/components/buy-sell/buy-sell.jsx

```diff
@@ -0,0 +1,105 @@
+import React     from 'react';
+import PropTypes from 'prop-types';
+import {
+    Button,
+    Dialog,
+    Loading }    from 'deriv-components';
+import Popup     from './popup.jsx';
+import           './buy-sell.scss';
+
+class BuySell extends React.Component {
+    state = {
+        buy_sell_list: [],
+        show_popup   : false,
+        type         : 'buy',
+        selected_ad  : {},
+        is_loading   : true,
```

Comment:
can you make this alphabetical?

Reply:
@oskar-binary will change the whole state name in his PR to match with the table, the only thing that popup needs is only the `selected_ad`.